omnigan/losses.py

```diff
@@ -408,22 +388,27 @@ def get_losses(opts, verbose, device=None):
         losses["G"]["p"]["reconstruction"] = ReconstructionLoss()
         losses["G"]["p"]["featmatch"] = FeatMatchLoss()
 
-    # task losses
-    # ? * add discriminator and gan loss to these task when no ground truth
-    # ?   instead of noisy label
-
+    # depth losses
     if "d" in opts.tasks:
-        losses["G"]["tasks"]["d"] = SIGMLoss(opts.train.lambdas.G.d.gml)
+        if opts.gen.d.loss == "data":
```

Comment:
Typo here, think you meant `dada`! But I actually also added this loss in my PR but called it `ReverseHuberLoss()` since it's the actual name of the loss that they use in the paper. Do you think it's maybe better to call it this way? opts.gen.d.loss could be `berhu ` or `sigm`.